app/views/Settings/ENDebugMenu.tsx

```diff
@@ -80,8 +78,20 @@ const ENDebugMenu = ({ navigation }: ENDebugMenuProps): JSX.Element => {
   };
 
   const handleOnPressToggleExposureNotifications = () => {
-    handleOnPressSimulationButton(BTNativeModule.toggleExposureNotifications)();
-    global.ExposureNotificationsOn = !global.ExposureNotificationsOn;
+    handleOnPressSimulationButton(toggleExposureNotifications)();
+  };
+
+  const showDebugVerificationCode = () => {
+    Alert.alert(
+      'Debug Verification Code:',
+      '111111',
```

Comment:
is this `111111` actually hooked up to anything or is it dummy data? Either why does it want to live in a constant?

Reply:
It's not hooked up atm. But good thinking, it should be stored in a constant. Would putting a `const DEBUG_VERIFICATION_CODE = '111111';` be the idiomatic way to do this?

Reply:
I would say yes.

constants that are configuration-ish in intent are usually are SCREAMING_SNAKE_CASE and are at the top of the file.

sometimes folk pull constants out to separate files, which i generally avoid as it makes it farther away from the thing that it is relevant to and is generally harder to maintain.